src/util/README.md

```diff
@@ -90,7 +90,7 @@ is initially populated from `irep_ids.def`, so for example the fourth entry
 in `irep_ids.def` is `“IREP_ID_ONE(type)”`, so the string “type” has index 3.
 You can refer to this \ref irep_idt as `ID_type`. The other kind of line you
 see is `“IREP_ID_TWO(C_source_location, #source_location)”`, which means the
-\ref irep_idt for the string “#source_location” can be referred to as
+\ref irep_idt for the string “\#source_location” can be referred to as
```

Comment:
Maybe this change doesn't actually achieve what it is supposed to do? Doxygen now appears to be complaining that the link request could not be satisfied.

Reply:
This changed worked.  The problem is the other #source_location on the previous line.  (I'm not sure how I missed this earlier!)

This has turned out to be much more tricky to fix - adding a \ doesn't work in this case (as explained in https://stackoverflow.com/a/36750674 ).  I haven't found a perfect solution to this, but have now committed something that works with slightly different double quotes.

Reply:
I think this is good, especially with the elaborate commit message. It's not beautiful, but anyone can easily figure out what's going on here.